drb/dockerscripts/rpmbuild-srcrpm-in-docker.sh

```diff
@@ -70,7 +70,7 @@ then
 		exit ${exitcode}
 	fi
 else
-    rpmbuild --rebuild ${RPMBUILD_OPTIONS} "${SRPMS_DIR}/${SRCRPM}" || { [ "bashonfail" == "$BASH_ON_FAIL" ] && { echo "Build failed, spawning a shell" ; /bin/bash ; exit 1; } || exit 1 ; }
+    eval rpmbuild --rebuild ${RPMBUILD_OPTIONS} "${SRPMS_DIR}/${SRCRPM}" || { [ "bashonfail" == "$BASH_ON_FAIL" ] && { echo "Build failed, spawning a shell" ; /bin/bash ; exit 1; } || exit 1 ; }
```

Comment:
Hello, could you explain me why would you choose to add an eval here? It seems quite arbitrary to me.